build.gradle

```diff
@@ -26,7 +26,7 @@ plugins {
   id "nebula.lint" version "17.1.1" apply false
   id "com.palantir.docker" version "0.28.0" apply false
   id "io.spring.dependency-management" version "1.0.11.RELEASE" apply false
-  id "org.ajoberstar.grgit" version "4.1.0" apply false
+  id "org.ajoberstar.grgit" version "4.1.1" apply false
```

Comment:
Should this PR be bumping a dependency version?

Reply:
I just did it to see if all tests pass. Anyways that failed still. Currently, the release team is looking into other alternative mirrors for this dependency. Will rebase once the fix is in.

Reply:
there is also already PR #7260 for this one-line change but it shouldn't conflict regardless of which is merged first or whether you rebase or not